test/calculation_test.exs

```diff
@@ -93,7 +93,7 @@ defmodule AshPostgres.CalculationTest do
       })
       |> Api.create!()
 
-    assert %{first_name_and_bob: "bob"} =
+    assert %{first_name_and_bob: "fred"} =
```

Comment:
I think this might actually not pass. I'm a little confused now how the test was initially passing.

Reply:
alright, fixed